internal/provider/schema_gitlab_application_settings.go

```diff
@@ -0,0 +1,3054 @@
+			Computed:    true,
+		},
+
+		"file_template_project_id": {
+			Description: "The ID of a project to load custom file templates from.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"first_day_of_week": {
+			Description: "Start day of the week for calendar views and date pickers. Valid values are 0 (default) for Sunday, 1 for Monday, and 6 for Saturday.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"geo_node_allowed_ips": {
+			Description: "Comma-separated list of IPs and CIDRs of allowed secondary nodes. For example, 1.1.1.1, 2.2.2.0/24.",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"geo_status_timeout": {
+			Description: "The amount of seconds after which a request to get a secondary node status times out.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"git_two_factor_session_expiry": {
+			Description: "Maximum duration (in minutes) of a session for Git operations when 2FA is enabled.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"gitaly_timeout_default": {
+			Description: "Default Gitaly timeout, in seconds. This timeout is not enforced for Git fetch/push operations or Sidekiq jobs. Set to 0 to disable timeouts.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"gitaly_timeout_fast": {
+			Description: "Gitaly fast operation timeout, in seconds. Some Gitaly operations are expected to be fast. If they exceed this threshold, there may be a problem with a storage shard and ‘failing fast’ can help maintain the stability of the GitLab instance. Set to 0 to disable timeouts.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"gitaly_timeout_medium": {
+			Description: "Medium Gitaly timeout, in seconds. This should be a value between the Fast and the Default timeout. Set to 0 to disable timeouts.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"grafana_enabled": {
+			Description: "Enable Grafana.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"grafana_url": {
+			Description: "Grafana URL.",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"gravatar_enabled": {
+			Description: "Enable Gravatar.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"hashed_storage_enabled": {
+			Description: "Create new projects using hashed storage paths: Enable immutable, hash-based paths and repository names to store repositories on disk. This prevents repositories from having to be moved or renamed when the Project URL changes and may improve disk I/O performance. (Always enabled in GitLab versions 13.0 and later, configuration is scheduled for removal in 14.0)",
```

Comment:
```suggestion
			Description: "Create new projects using hashed storage paths: Enable immutable, hash-based paths and repository names to store repositories on disk. This prevents repositories from having to be moved or renamed when the Project URL changes and may improve disk I/O performance. (Always enabled in GitLab versions 13.0 and later, configuration is scheduled for removal in 14.0).",
```

Reply:
Also, while this one isn't flagged explicitly as deprecated, 14.10 is _barely_ outside the range of what we've talked about supporting in the past. I'm good with leaving this one hear since folks are undoubtedly still using the provider for older GitLab versions, but feels like someone keeping their provider very up-to-date would also be keeping GitLab equally up-to-date.

Thoughts here?

Reply:
Yes, again - while I agree I'd still leave it in for now and remove once it's gone upstream - just to remove my effort here a little bit :)